app/src/main/java/com/github/swent/echo/compose/map/LocationSelector.kt

```diff
@@ -0,0 +1,56 @@
+package com.github.swent.echo.compose.map
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import com.github.swent.echo.R
+import com.mapbox.mapboxsdk.geometry.LatLng
+import org.ramani.compose.CameraPosition
+import org.ramani.compose.Circle
+import org.ramani.compose.MapLibre
+import org.ramani.compose.Symbol
+
+@Composable
+fun LocationSelector(
+    modifier: Modifier = Modifier,
+    initialLocation: LatLng = MAP_CENTER.toLatLng(),
+    locationCallback: (LatLng) -> Unit
+) {
+    val context = LocalContext.current
+    val styleUrl =
+        context.getString(R.string.maptiler_base_style_url) +
+            context.getString(R.string.maptiler_api_key)
+    var markerCenter by remember { mutableStateOf(initialLocation) }
+    var mapCenter by remember { mutableStateOf(initialLocation) }
+    MapLibre(
+        modifier = modifier.testTag("location-selector-map-libre"),
+        styleUrl = styleUrl,
+        cameraPosition = CameraPosition(target = mapCenter, zoom = DEFAULT_ZOOM)
```

Comment:
The way the event screen currently works is that the values of the `Event` object being edited are used. This means that to have a specific "default" location, the `EMPTY` object for the `Location` data class should probably use `MAP_CENTER` instead of `(0.0, 0.0)`. 
I'm not sure how we would go about using the GPS. In my mind, the way events should be created is by selecting the location first, which then would be passed to the current event creation screen, but that would need a fair amount of work.